core/src/main/java/nu/yona/server/crypto/CryptoUtil.java

```diff
@@ -25,7 +25,7 @@ public static String getRandomString(int length)
 		return randomString.substring(0, Math.min(length, randomString.length()));
 	}
 
-	static SecureRandom getSecureRandomInstance()
+	public static SecureRandom getSecureRandomInstance()
```

Comment:
Make this private again and add a method ``int generateRandomNumber(int digits)`` instead.